vspace/src/main/webapp/WEB-INF/views/accessdenied.jsp

```diff
@@ -0,0 +1,11 @@
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+
+
+<div style="padding-top: 60px;">
+
+<h1>You dont seem to have the authority to access the requested page!</h1>
+<h2><a href="<c:url value="/staff/dashboard"/>">Go to dashboard</a></h2>
+</div>
```

Comment:
Thinking about this, make this page use the exhibition layout and don't provide a link to the dashboard. This page will always be shown, e.g. when a visitor of the exhibition tries to access a restricted page. If they are not logged in they should not see the staff area layout or a link to the dashboard.

Reply:
Let me correct that, if a user has an account but doesn't not have staff or admin rights, they should not necessarily see the staff area layout.